src/Command/RunCommand.php

```diff
@@ -339,21 +324,48 @@ private function getComponent(ClientWrapper $clientWrapper, string $id): array
     private function resolveVariables(
         ClientWrapper $clientWrapper,
         array $jobDefinitions,
+        ?string $branchId,
         ?string $variableValuesId,
         array $variableValuesData
     ): array {
+        if ($variableValuesId === '') {
+            throw new InvalidArgumentException('$variableValuesId must not be empty string');
+        }
+
+        if ($branchId === null || $branchId === 'default') {
```

Comment:
Tohleto bych potřeboval strčit ještě před tohle https://github.com/keboola/job-runner/pull/219/files#diff-b05a137a6323c6e93f4fe4c9bb8babe4de1e6c5e6a8a98c1b43073d62046bd68L203 a vrazit to do clientOptions
protože pak se konečně dostanem k tomu o čem jsem mluvil ve st - že to branchId je ve zpracování jobu resolvnutý a všude stejně (tzn. konfigurace a variables z vaultu budeš číst se stejným branchId) a pak už bude "jenom" stačit to celý potáhnout vejš do job resolveru

Reply:
Ten link me na nic nevede, pred co by is to potreboval strcit?

Reply:
https://github.com/keboola/job-runner/blob/f02496f3f0a326e2d4a3d6e754e8403919f9f2c6/src/Command/RunCommand.php#L187

Reply:
Chapu (asi :smile:), pripravim to samostatne